docs/conf.py

```diff
@@ -30,19 +30,14 @@
 from docutils.utils import get_source_line
 
 
-def _warn_node(self, msg, node):
-    """Do not warn on external images."""
-    if not msg.startswith('nonlocal image URI found:'):
-        self._warnfunc(msg, '{0}:{1}'.format(get_source_line(node)))
-
-sphinx.environment.BuildEnvironment.warn_node = _warn_node
-
-
 # -- General configuration ------------------------------------------------
 
 # If your documentation needs a minimal Sphinx version, state it here.
 #needs_sphinx = '1.0'
 
+# Do not warn on external images.
```

Comment:
👍 Can you make a PR to [inveniosoftware/cookiecutter-invenio-module](https://github.com/inveniosoftware/cookiecutter-invenio-module/blob/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/docs/conf.py) with this change as well.